scrapy_po/middleware.py

```diff
@@ -21,45 +23,44 @@ class InjectPageObjectsMiddleware:
     """
     @inlineCallbacks
     def process_response(self, request: Request, response, spider):
-        # find out which arguments need to be created
+        # find out the dependencies
         callback = get_callback(request, spider)
-        kwargs_to_provide = andi.to_provide(callback,
-                                            can_provide=self.is_page_object)
+        providers = build_providers(response)
```

Comment:
provider_instances? PyCharm shows me that this name shadows "providers" imported from page_input_providers, which is not good

Reply:
Good catch. Changed here: https://github.com/scrapinghub/scrapy-po/pull/5/commits/9983a04d037b612d51c887dd3e7db9a3a0f5c32a